src/components/Accordion/Accordion.test.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { mount } from 'enzyme';
+
+import { Accordion, AccordionContent } from '../../index';
+
+const isFixed = true;
+
+const testAccordion = props => (
+  <Accordion height={200} accordionId="test" expandedKey="1">
+    <AccordionContent title="Panel 1" eventKey="1" fixed={isFixed}>
+      <div>test blahs</div>
+    </AccordionContent>
+  </Accordion>
+);
+test('fixed accordion renders', () => {
+  const component = mount(testAccordion());
+  expect(component.render()).toMatchSnapshot();
+});
+test('Accordion renders', () => {
+  const component = mount(
```

Comment:
can we use `shallow` instead of `mount` for tests that do not need interaction? see [Wizard tests](https://github.com/patternfly/patternfly-react/blob/master/src/components/Wizard/Wizard.test.js) for example